app/metrics.yaml

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+no_lint:
+  - CATEGORY_GENERIC
+
+metrics:
+  default_browser:
+    type: boolean
+    lifetime: application
+    description: |
+      Is Focus the default browser?
```

Comment:
It would be important, for folks consuming the data documentation when analysing the data, understanding _when_ this is detected (right after startup? What happens if the default is changed while the browser is running?)

Reply:
Fair point. Focus seems to do this based on the user changing the default browser through app settings alone.. which seems wrong.

I took this metric from Fenix and that seems to be querying the result correctly (i.e. from the OS settings), so I think our metrics for Focus might have been off for this pref. 😅

Reply:
Awesome :) Good catch!